README.adoc

```diff
@@ -92,7 +92,7 @@ In order to use private image on quay a pull secret need to be crated:
 [source,shell]
 ----
 kubectl create secret generic addon-pullsecret \
-    --from-file=${path of rhoas-pull-docker} \
+    --from-file=.dockerconfigjson=${path of rhoas-pull-docker} \
```

Comment:
that's ok, but then please add .dockerconfigjson to the gitignore as I'm pretty sure it would be leaked

Reply:
This only sets the key name inside the kubernetes secret - it does not actually create the file locally.

Reply:
oh sorry, I miread it